include/zephyr/bluetooth/bluetooth.h

```diff
@@ -812,7 +874,8 @@ enum {
 	 *
 	 * @note Cannot be set if BT_LE_ADV_OPT_CODED is set.
 	 *
-	 * @note Requires @ref BT_LE_ADV_OPT_EXT_ADV.
+	 * @note Requires @ref BT_LE_ADV_OPT_EXT_ADV bit to be set in @ref advertising_options
+	 * enum.
```

Comment:
Generally missing for all these bits:
```suggestion
	 * @note Requires @ref BT_LE_ADV_OPT_EXT_ADV bit to be set in the @ref advertising_options
	 * enum.
```

Could also replace `enum` with `field`. It doesn't really matter if it's an enum of another data type